jgrapht-ext/src/main/java/org/jgrapht/ext/StringUtility.java

```diff
@@ -0,0 +1,26 @@
+package org.jgrapht.ext;
+
+public class StringUtility
+{
+    /**
+     * Removes quotes from a string
+     * 
+     * @param in a quoted string
+     * @return the string without enclosing quotes; if <code>in</code> is not
+     *         quoted, the original string is returned.
+     */
+    public static String deQuote (String in)
```

Comment:
It is highly likely to use in the future the commons-lang3 dependency as it is also needed elsewhere. However, until we decide to do so, place this one as a private static in the `GmlImporter`.